gen/slice_of.go

```diff
@@ -7,11 +7,13 @@ import (
 )
 
 // SliceOf generates an arbitrary slice of generated elements
+// genParams.MaxSize sets an (exclusive) upper limit on the size of the slice
+// genParams.MinSize sets an (inclusive) lower limit on the size of the slice
 func SliceOf(elementGen gopter.Gen) gopter.Gen {
 	return func(genParams *gopter.GenParameters) *gopter.GenResult {
 		len := 0
-		if genParams.Size > 0 {
-			len = genParams.Rng.Intn(genParams.Size)
+		if genParams.MaxSize > 0 || genParams.MinSize > 0 {
+			len = genParams.Rng.Intn(genParams.MaxSize-genParams.MinSize) + genParams.MinSize
```

Comment:
I think there should be a check that genParams.MaxSize > genParams.MinSize ...

Not sure what happens is Intn gets a negative parameter